cfgov/unprocessed/css/video-player.less

```diff
@@ -55,6 +55,8 @@
     // @black not used because it's not totally black.
     background-color: #000;
     text-align: center;
+    height: 100%;
+    width: 100%;
```

Comment:
We should be sure these things don't break the video player in other places. Check an event page and a story page.